Gems/Multiplayer/Assets/TranslationAssets/Classes/NetworkTransformComponentBase.names

```diff
@@ -0,0 +1,117 @@
+            "methods": [
+                {
+                    "base": "MultiplayerTeleportByEntityId",
+                    "entry": {
+                        "name": "In",
+                        "tooltip": "Deprecated, please use `Multiplayer Teleport` instead"
+                    },
+                    "exit": {
+                        "name": "Out",
+                        "tooltip": "Deprecated, please use `Multiplayer Teleport` instead"
+                    },
+                    "details": {
+                        "name": "MultiplayerTeleportByEntityId"
+                    }
+                },
+                {
+                    "base": "SetResetCount",
+                    "entry": {
+                        "name": "In",
+                        "tooltip": "When signaled, this will invoke Set Reset Count"
+                    },
+                    "exit": {
+                        "name": "Out",
+                        "tooltip": "Signaled after Set Reset Count is invoked"
+                    },
+                    "details": {
+                        "name": "Set Reset Count"
+                    },
+                    "params": [
+                        {
+                            "typeid": "{6383F1D3-BB27-4E6B-A49A-6409B2059EAA}",
+                            "details": {
+                                "name": "Entity Id",
+                                "tooltip": "Entity Unique Id"
+                            }
+                        },
+                        {
+                            "typeid": "{72B9409A-7D1A-4831-9CFE-FCB3FADD3426}",
+                            "details": {
+                                "name": "unsigned char"
+                            }
+                        }
+                    ]
+                },
+                {
+                    "base": "MultiplayerTeleport",
+                    "entry": {
+                        "name": "In",
+                        "tooltip": "When signaled, this will invoke Multiplayer Teleport"
+                    },
+                    "exit": {
+                        "name": "Out",
+                        "tooltip": "Signaled after Multiplayer Teleport is invoked"
+                    },
+                    "details": {
+                        "name": "Multiplayer Teleport"
+                    },
+                    "params": [
+                        {
+                            "typeid": "{6383F1D3-BB27-4E6B-A49A-6409B2059EAA}",
+                            "details": {
+                                "name": "Source",
+                                "tooltip": "The Source containing the NetworkTransformComponentController"
+                            }
+                        },
+                        {
+                            "typeid": "{8379EB7D-01FA-4538-B64B-A6543B4BE73D}",
+                            "details": {
+                                "name": "teleport To Position"
```

Comment:
Shouldn't this be capital "Teleport To Position"?

Reply:
Done